hugegraph-core/src/main/java/com/baidu/hugegraph/config/CoreOptions.java

```diff
@@ -115,6 +115,15 @@ public static synchronized CoreOptions instance() {
                     10L
             );
 
+    public static final ConfigOption<Long> CONNECTION_DETECT_INTERVAL =
+            new ConfigOption<>(
+                    "store.connection_detect_interval",
+                    "The interval for detecting connection, if connection's " +
+                    "idle time exceed this interval, detect it before using.",
```

Comment:
The interval in seconds for detecting connections, 
if the idle time of a connection exceeds this value, 
detect it and reconnect if needed before using,
value 0 means detecting every time.